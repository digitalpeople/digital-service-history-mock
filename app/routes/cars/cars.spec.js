import {
  getCars,
  getCarsById,
} from './cars';

jest.mock('./cars-handler', () => ({
  getCarsHandler: 'getCarsHandlerMockFunction',
  getCarsByIdHandler: 'getCarsByIdHandlerMockFunction',
}));

describe('getCars', () => {
  it('should app.get with correct path', () => {
    expect.assertions(1);

    const app = {
      get: jest.fn(),
    };

    getCars(app);

    expect(app.get.mock.calls[0]).toMatchSnapshot();
  });
});

describe('getCarsById', () => {
  it('should app.get with correct path', () => {
    expect.assertions(1);

    const app = {
      get: jest.fn(),
    };

    getCarsById(app);

    expect(app.get.mock.calls[0]).toMatchSnapshot();
  });
});
