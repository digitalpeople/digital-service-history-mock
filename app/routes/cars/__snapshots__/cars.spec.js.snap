// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCars should call app.get with correct path 1`] = `
Array [
  "/cars/",
  "getCarsHandlerMockFunction",
]
`;

exports[`getCarsById should call app.get with correct path 1`] = `
Array [
  "/cars/:id/",
  "getCarsByIdHandlerMockFunction",
]
`;
